swim/member.go

```diff
@@ -204,6 +206,11 @@ func (c *Change) populateSource(m *Member) {
 	c.SourceIncarnation = m.Incarnation
 }
 
+func (c *Change) scrubSource() {
```

Comment:
What if we populate more "source" fields in the future? Currently there's no "safeguard" against forgetting to clear it here. Maybe at least add a comment to `populateSource`?
Bonus points for a test that:
- creates a change -> `popuplateSource` -> `scrubSource` -> compare with an new/empty change.